packages/markdown/src/markdown.ts

```diff
@@ -82,8 +84,15 @@ export const createMarkdown = ({
   }
 
   // process code fence
-  if (code !== false) {
-    md.use<CodePluginOptions>(codePlugin, code)
+  if (code) {
+    logger.warn(
```

Comment:
Have we updated the docs? Or maybe we need to link to plugin docs?

Reply:
No, I will open a pr for this, note a route related pr is open a few days ago not not being merged.